src/modules/samples/rust/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+use libc::{c_int, c_char ,c_void};//, c_void
+use std::ffi::CStr;
+use std::ffi::CString;
+use std::ptr::NonNull;
+
+mod sample;
+
+// Constants and Method headers were generated by bindgen
+// (https://github.com/rust-lang/rust-bindgen)
+pub const MMI_OK: u32 = 0;
+pub type MMI_HANDLE = *mut ::std::os::raw::c_void;
+pub type MMI_JSON_STRING = *mut ::std::os::raw::c_char;
+
+// MSFT change: Added the #[no_mangle] annotation and defined the methods
+#[no_mangle]
+pub extern "C" fn MmiGetInfo(
+        clientName: *const ::std::os::raw::c_char,
```

Comment:
Do these methods (MmiGetInfo, MmiOpen, etc.) need to be formatted like this?

Reply:
Is it rust convention to space arguments like this?

Reply:
Also, it looks like it is indented 8 spaces, if keeping the arguments indented, please use 4 spaces